packages/market-service/src/coingecko/coingecko.test.ts

```diff
@@ -93,6 +95,15 @@ describe('coingecko market service', () => {
       expect(Object.keys(result).length).toEqual(0)
     })
 
+    it('does not get rate limited', async () => {
```

Comment:
Discussed on Discord, not sure about the value of this since we're testing against some scenario (rate-limiting) that would never happen in tests.
Proposed to implement some poor man's rate limiting of mocks (send 429 if within rate limit, or response if not limited, checked depending on the numbers of axios.get calls per 1000ms) in tests if it's doable within reasonable effort, but I'm fine without it.

In this case, I'd vouch for removing the tests from that diff, but let's wait on another reviewers before doing so.

Reply:
I've redone the tests now